setup.py

```diff
@@ -30,62 +30,60 @@ def __init__(self, *args, **kwargs):
         egg_info.__init__(self, *args, **kwargs)
         try:
             import setuptools_scm.integration
+
             setuptools_scm.integration.find_files = lambda _: []
         except ImportError:
             pass
 
 
-with open('README.rst') as f:
+with open("README.rst") as f:
     README = f.read()
 
 setup(
-    name='bluepysnap',
+    name="bluepysnap",
     python_requires=">=3.7",
     install_requires=[
-        'cached_property>=1.0',
-        'h5py>=3.0.1,<4.0.0',
-        'libsonata>=0.1.6,<1.0.0',
-        'morphio>=3.0.0,<4.0.0',
-        'morph-tool>=2.4.3,<3.0.0',
-        'numpy>=1.8,<2.0.0',
-        'pandas>=1.0.0,<2.0.0',
-        'click>=7.0',
-        'more-itertools>=8.2.0'
+        "cached_property>=1.0",
+        "h5py>=3.0.1,<4.0.0",
+        "libsonata>=0.1.6,<1.0.0",
+        "morphio>=3.0.0,<4.0.0",
+        "morph-tool>=2.4.3,<3.0.0",
+        "numpy>=1.8,<2.0.0",
+        "pandas>=1.0.0,<2.0.0",
+        "click>=7.0",
+        "more-itertools>=8.2.0",
     ],
     extras_require={
-        'docs': ['sphinx', 'sphinx-bluebrain-theme'],
-        'plots': ["matplotlib>=3.0.0,<4.0.0"],
+        "docs": ["sphinx", "sphinx-bluebrain-theme"],
+        "plots": ["matplotlib>=3.0.0,<4.0.0"],
     },
     packages=find_packages(),
     use_scm_version=True,
     setup_requires=[
-        'setuptools_scm',
+        "setuptools_scm",
     ],
     cmdclass={
-        'egg_info': EggInfo,
+        "egg_info": EggInfo,
```

Comment:
Irrelevant to this review, but a static way to do this would be to explicitly exclude what does not need to be part of the dist in the `MANIFEST.in`

Reply:
Cool, always happy to add improvements 👍